### DISCUSSION TITLE
Trouble with dead but enabled services, sshd, ha-proxy

### DISCUSSION THREAD

**Discussion by rs-development** [tteck/Proxmox#554] [Help Needed]
For some reason I need to enable the ssh service manually after a Debian/Photoprism/Docker LXC restart.

After reboot I have:

```
* ssh.service - OpenBSD Secure Shell server
     Loaded: loaded (/lib/systemd/system/ssh.service; enabled; vendor preset: enabled)
     Active: inactive (dead)
       Docs: man:sshd(8)
             man:sshd_config(5)
```

Running `systemctl start ssh.service` will start the ssh service and login works fine untill the next reboot.

While testing this I also found that the problem spread to other services like HA-Proxy. I have multiple Containers with the same issue, while I don't think this is necessarily an issue with your wonderful scripts, I wonder if I am the only one? 

Things that all containers have in common for me is that I install zsh,git,omz,curl,wget,htop,bmon but i don't see any relation here.

**Comment by tteck** [comment]
Did you enable root ssh access to each LXC?

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/385

**Reply by rs-development** [reply]
Yes i have enabled root access, but not with password since I only use keys for SSH.

So i have checked the journal and figured out that ha-proxy was not starting for 5-minutes since proxmox did not remove old network interfaces completely so ipupdown was not done etc.

I assume ssh.service is not starting since it requires `After=network.target auditd.service` 
Auditd is not installed per default.

**Comment by rs-development** [comment]
It was probably me, sorry for bothering.

I somehow enabled the ssh.socket service AND ssh.service simultaneously. That was obviously not working, after disabling that everything works as expected.
